tools/charm_module_patches/src/Parallel/Algorithms/AlgorithmArray.def.h.patch

```diff
@@ -0,0 +1,49 @@
+Distributed under the MIT License.
+See LICENSE.txt for details.
+--- AlgorithmArray.def.h	2021-04-01 14:42:23.928094971 -0700
++++ alterations/AlgorithmArray.def.h	2021-04-01 14:44:19.599096051 -0700
+@@ -308,7 +308,7 @@
+ {
+   ckCheck();
+   AlgorithmArray <ParallelComponent, SpectreArrayIndex>  *obj = ckLocal();
+-  if (obj) {
++  if (obj != nullptr and not Parallel::detail::max_inline_entry_methods_reached()) {
```

Comment:
I am confused; above you removed a similar conditional but there was an else branch; what happens now when `max_inline_entry_methods_reached()` is true?

Reply:
the charm generated functions already have a message-passing `else` (that's below the patch range) that executes when the object isn't on the current core. This just ensures that it also does the message-passing thing if it hits our inline cap.

Reply:
the charm generated functions already have a message-passing `else` (that's below the patch range) that executes when the object isn't on the current core. This just ensures that it also does the message-passing thing if it hits our inline cap.

Reply:
okay, thanks